### DISCUSSION TITLE
Matek F722SE Can't Connect to MAMBA F60_BL32 4in1 ESC

### DISCUSSION THREAD

**Discussion by jasyohuang** [iNavFlight/inav#8254] [Q&A]
Hi, i'm having trouble with my quad build where the esc can't communicate with the FC (Inav 5.0.0). When i powered up the ESC, only 3 beeps is heard. i checked the 5v regulator with multimeter (5v pin and ground pin) and it works properly showing 5volt on multimeter readings. i connected the FV to S1 to S4 pins and current ping from the esc to current pin on the FC. is this some kind of firmware issue ?

**Comment by jasyohuang** [comment]
the ESC is detected through BL_Heli configurator though

**Comment by JulioCesarMatias** [comment]
Something new?
